docs/docs/configuring-usage-with-plugin-options.md

```diff
@@ -54,24 +54,6 @@ The following table lists possible options values and an example plugin that mak
 
```

Comment:
Upon rereading this doc, it might make more sense for the "what can be passed in as options" section to come before the "where to access plugin options" section. So that it flows from why you may want plugin options → what kinds of options you can use → how to set up those options.